junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java

```diff
@@ -191,6 +204,23 @@ public boolean supportsParameter(ParameterContext parameterContext, ExtensionCon
         }
     }
 
+    /**
+     * @see org.junit.jupiter.engine.extension.TestInfoParameterResolver#supportsParameter
+     * @see org.junit.jupiter.engine.extension.RepetitionExtension#supportsParameter
+     * @see org.junit.jupiter.engine.extension.TestReporterParameterResolver#supportsParameter
+     * @see org.junit.jupiter.engine.extension.TempDirectory#supportsParameter
+     */
+    private boolean isJUnitResolvedParameter(ParameterContext parameterContext, ExtensionContext extensionContext) {
```

Comment:
The `ExtensionContext` parameter is never used.

Reply:
agree and removed